docs/reference/setup/configuration.asciidoc

```diff
@@ -71,14 +71,21 @@ path.logs: /var/log/elasticsearch
 
 Environment variables referenced with the `${...}` notation within the
 configuration file will be replaced with the value of the environment
-variable, for instance:
+variable. For example:
 
 [source,yaml]
 --------------------------------------------------
 node.name:    ${HOSTNAME}
 network.host: ${ES_NETWORK_HOST}
 --------------------------------------------------
 
+When specifying values for environment variables, you can only use strings. More complex data structures like lists are not supported. However, settings that accept data structures such as lists also accepted comma-separated strings. For example, you can specify multiple values for the `${HOSTNAME}` variable from the previous example:
+
+[source,console]
+--------------------------------------------------
+export HOSTNAME=“host1”, “host2"
+--------------------------------------------------
```

Comment:
This should be `[source,yaml]`. `[source,console]` snippets get tested by our CI, which would fail here. This is also just a bit of YAML, not a Kibana console snippet.

You only need four `-` characters to denote a snippet. I wouldn't do more than that.
```suggestion
[source,yaml]
----
export HOSTNAME=“host1”, “host2"
----
```